specification/keyvault/resource-manager/readme.md

```diff
@@ -38,6 +38,7 @@ These settings apply only when `--tag=package-2019-09` is specified on the comma
 input-file:
 - Microsoft.KeyVault/stable/2019-09-01/keyvault.json
 - Microsoft.KeyVault/stable/2019-09-01/providers.json
+- Microsoft.KeyVault/stable/2019-09-01/keys.json
```

Comment:
Are you sure that all generators are pointing to this readme.md for configuration? At least for data plane, we point at the JSON files directly. I.e. this should be merged into `keyvault.json` to work with existing tooling and configuration.

Reply:
Thanks for the feedback - this is my first time authoring a swagger spec. I'll look into reverting this and updating `keyvault.json` instead. (I wanted to avoid updating any existing files and only add new files, so as to avoid any breaking changes.)